zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SfvChecksumImpl.java

```diff
@@ -44,14 +48,19 @@ final class SfvChecksumImpl implements MutableChecksumsSFV {
   private static final String FORMAT_SNAPSHOT_DIRECTORY_LINE = "; snapshot directory = %s\n";
   private static final String FORMAT_FILE_CRC_LINE = "%s   %s\n";
   private static final String FORMAT_COMBINED_VALUE_LINE = "; combinedValue = %s\n";
+  private static final String FORMAT_CHECKSUM_VERSION_LINE = "; checksumVersion = %s\n";
   private static final String FILE_CRC_SEPARATOR_REGEX = " {3}";
+  private static final Pattern FILE_VERSION_PATTERN =
+      Pattern.compile(
+          "; checksumVersion = (" + SnapshotVersion.MANUAL_CHECKSUM + "|" + CHECKSUM_FROM_DB + ")");
```

Comment:
:x: This hardcodes the set of versions. If we continue to extend this, older brokers will not recognize newly added `checksumVersion` variants and thus assume that no version is set (because the pattern doesn't match). This is wrong because an old broker should be able to detect that `checksumVersion` is something new and not supported yet.

Reply:
Yep will fix this

Reply:
fixed